### ISSUE TITLE
awslogs streams <group> displays nothing in some situations

### ISSUE THREAD

**Issue by wayne-luminal** [jorgebastida/awslogs#57]
If you have a log group that contains streams but those streams were not written to before current_time+5m as calculated by `parse_datetime()`, the streams command will not display them. I would expect `awslogs streams <group>` to display all the streams inside the group regardless of when they were last written to.

**Comment by nathanleiby** [comment]
Agreed - just saw this behavior and it also seemed strange to me. It should be more explicit about the time filtering in `awslogs streams`, or not do it by default

**Comment by redterror** [comment]
Piling on, out of the box I thought things were broken until I found this.  My expectation was that `streams` would show me all available streams, regardless of recency.

**Comment by chashi** [comment]
To make what's already been identified in the comments above painfully obvious, the workaround is to add --start. For example:

`awslogs streams GROUP --start "5 day ago"`

**Comment by jesstelford** [comment]
@chashi Thanks for the workaround - I had no idea why my streams weren't showing even after reading previous comments!

**Comment by whereisaaron** [comment]
Glad I found this, I was coming here to file a bug report, that 'awslog groups' wasn't listing all my groups.
